scoresheets/WhereIsThis.tex

```diff
@@ -0,0 +1,15 @@
+
+The maximum time for this test is 10 minutes.
+
+\begin{scorelist}
+	\scoreheading{Main Goal}
+	\scoreitem[10]{100}{Describe or show the requested location accurately.}
```

Comment:
Logistically unfeasible. Is there any way to reduce it to 5 and make each more challenging?

Reply:
I kind of liked the idea to have lots of interactions. Because that is one of the few ways to get some objectivity into HRI. But yeah getting 10 volunteers might be difficult. I'll think about this